secretmanager/snippets/regional_samples/create_regional_secret.py

```diff
@@ -0,0 +1,93 @@
+#!/usr/bin/env python
+
+# Copyright 2024 Google LLC
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+"""
+command line application and sample code for creating a new regional secret.
+"""
+
+import argparse
+from typing import Optional
+
+from google.cloud import secretmanager_v1
+
+
+# [START secretmanager_v1_create_regional_secret]
+def create_regional_secret(
+    project_id: str, location_id: str, secret_id: str, ttl: Optional[str] = None
+) -> secretmanager_v1.Secret:
+    """
+    Create a new regional secret with the given name. A secret is a logical wrapper
+    around a collection of secret versions. Secret versions hold the actual
+    secret material.
```

Comment:
```suggestion
    Creates a new regional secret with the given name.
    
    A secret is a logical wrapper around a collection of secret versions. Secret versions
    hold the actual secret material.
```

There should be a one short summary, followed by a new paragraph of supplemental information as required.